src/components/Card.spec.js

```diff
@@ -0,0 +1,24 @@
+import Card from './Card';
+import { render, screen } from '@testing-library/react';
+
+describe('Card', () => {
+  it('renders a word and example and/or explanation', () => {
```

Comment:
I would suggest you remove the part `/or`, because your testing the explanation 🙂

Reply:
thank you!